orekit/src/orekit_orbit.py

```diff
@@ -1,23 +1,19 @@
 """ Simulate Orekit orbit as poliastro one """
 
-import os
-
 import numpy as np
 from astropy import units as u
-from orekit.pyhelpers import setup_orekit_curdir
-from org.hipparchus.geometry.euclidean.threed import Vector3D
-from org.orekit.frames import FramesFactory
-from org.orekit.orbits import KeplerianOrbit, OrbitType, PositionAngle
-from org.orekit.propagation.analytical import KeplerianPropagator
-from org.orekit.time import AbsoluteDate
-from org.orekit.utils import Constants, PVCoordinates
+# Start orekit virtual machine, load data and make flake8 to ignore this line
 from utils import setup_orekit_env
```

Comment:
On second thought: do you see any disadvantage on doing

```suggestion
from utils import vm
```

and make `utils.py` call

```
vm = setup_orekit_env()
```

at the end? The advantages of this approach are:

1. Python modules act like a singleton, therefore we have a single `vm` for everyone importing the module
2. We avoid the `# noqa: E402`

Reply:
(It `utils.py` contains more stuff, perhaps a dedicated module could be created only for this)

Reply:
Oh! I see... I completely agree with this approach, let me check if it works as expected.

Reply:
Previous approach simplifies a lot, but `noqa: F401` is still required. Nevertheless, code became more clean, let me push the changes for you to check.

Reply:
Notice that the `# isort: split` is required as `from utils import vm` statement needs to be before `from org.` ones. However, `flake8` complains about the non-usage of `vm` variable. I've tried to add something like `# noqa isort: split` and similar combinations, but only on of the tools (flake8 and isort) ignores the line, not both at at the same time...

Reply:
Alright, everything should be fixed now. Although one inline comment is required, everything looks more "natural".

Reply:
I see! If the `vm` variable is not used, perhaps `import utils` with a comment explaning what happens should be enough. However, it's good enough this way :)